plugins/catalog-backend/src/next/NextRouter.ts

```diff
@@ -59,6 +70,14 @@ export async function createNextRouter(
     logger.info('Catalog is running in readonly mode');
   }
 
+  if (processingEngine) {
+    router.post('/refresh', async (req, res) => {
+      const refreshOptions: CatalogProcessingEngineRefreshOptions = req.body;
+      await processingEngine.refresh(refreshOptions);
+      res.status(200).send();
```

Comment:
Should we return the new entity here or no?

Reply:
Actually, that would be quite interesting and something we thought too

Reply:
Discussed that for now it's a little bit of scope creep as we don't currently wait until the entity has finished. Will follow up later but for MVP we will just have a `refresh queued` notification in the frontend when you run it.

Reply:
Yes, and I think this one will turn out to be a batch or query based endpoint too - so the refreshed items may be a long list as well

Reply:
Having a way of knowing that the import is completed was always an issue for us, but we already implemented a workaround, so it's not that important here anymore (we just query the catalog till the expected items are available 😆 )